packages/client/src/App.js

```diff
@@ -3,6 +3,7 @@ import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
 import './App.css';
 import { LandingPage } from './containers/LandingPage/LandingPage.Container';
 import { PageNotFound } from './containers/PageNotFound/PageNotFound.Container';
+import ViewPage from './containers/ViewPage/ViewPage';
```

Comment:
I see there is an inconsistency in how other imports are being called and this one: take a look. Other imports are not `default` exports so they are wrapped in `{}`. Also the file naming is slightly different.

Protip: maintain consitency in naming files and exports across the team. If you don't agree, team should align on one naming strategy that satisfies everyone :)

Reply:
Thx for your advice!Yeah,we have a discussion today and I will improve it